.travis.yml

```diff
@@ -342,4 +355,38 @@ matrix:
             | ( grep '.*[\\|\/]PinNames.h$' || true ) \
             | while read file; do python ./hal/tests/pinvalidate/pinvalidate.py -vfp "${file}"; done
         - git diff --exit-code --diff-filter=d --color
+    - <<: extended-cmakecheck
+      stage: "Extended"
+      name: "cmakecheck"
+      env: NAME=cmakecheck
+      language: python
+      python: 3.8
+      install:
+        # Hide Travis-preinstalled CMake
+        - sudo rm -rf /usr/local/cmake*
```

Comment:
Do we need this second hiding here or was the first "Extended" hiding OK?

Reply:
The "Extended" hiding didn't work the first time because I hadn't added sources to the apt config, it should work now though.

Reply:
@0xc0170 does it make sense to create a new "CMake" job in our config?